subsys/net/lib/nrf_cloud/src/nrf_cloud_fsm.c

```diff
@@ -37,7 +37,6 @@ static const fsm_transition idle_fsm_transition[NCT_EVT_TOTAL] = {
 
 static const fsm_transition initialized_fsm_transition[NCT_EVT_TOTAL] = {
 	[NCT_EVT_CONNECTED] = connection_handler,
-	[NCT_EVT_DISCONNECTED] = disconnection_handler,
```

Comment:
@simensrostad you added this line in https://github.com/nrfconnect/sdk-nrf/pull/3472 

i'm removing because it should not be necessary to handle a disconnection if the device is in state: `STATE_INITIALIZED`.

can you please confirm. 
i'm not sure if there is some case that i'm missing.

Reply:
I don't exactly remember why I added this, but I think it was related to the MQTT DISCONNECT event not being forwarded to the application in case some corner case network issue occurred. I'll do some testing after the release to figure out why I added it.

Reply:
Cannot find any reason why this should be kept.